### PR TITLE
Build on mac

### PR DESCRIPTION
**Pull request by organom**
Added install script for mac and make build and make clean as helpers for development.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+default:
+	@echo "No option defined"
+
+.PHONY: build
+build:
```
Comment (ilyash-b) [inline Makefile]: Can't call `build` a target that installs. Should be `install-linux` ?

File: install-mac.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+brew bundle --file=- <<-EOS
```
Comment (ilyash-b) [inline install-mac.sh]: Why not use `Brewfile`?

File: install-mac.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+brew "libffi"
+brew "peg"
+brew "cmake"
+brew "pandoc"
+brew "awk"
+brew "make"
+EOS
+
+source build.macos.env.sh
+make build
```
Comment (ilyash-b) [inline install-mac.sh]: Should be `make install-mac`?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+	( mkdir -p build && cd build && cmake .. && make && sudo make install )
+clean:
+	rm -rf build
+
+install-mac:
```
Comment (ilyash-b) [inline Makefile]: Currently unused.

File: install-mac.sh (hunk 2 above)
Comment (organom) [reply install-mac.sh]: dont remember why i was doing this via brewfile anymore. left if as normal brew install commands. Anything you can remember that would make us needed the brewfile?

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,18 @@
+default:
+	@echo "No option defined"
```
Comment (ilyash-b) [inline Makefile]: option -> target

**Pull request merged**